spec/spec_helper.rb

```diff
@@ -36,4 +36,8 @@
   # the seed, which is printed after each run.
   #     --seed 1234
   config.order = "random"
+
+  config.expect_with :rspec do |c|
+    c.syntax = [:should, :expect]
```

Comment:
Have you considered using https://github.com/yujinakayama/transpec to convert our code to use the expect syntax?